apps/design-land/src/app/foundations/color/color.component.html

```diff
@@ -124,3 +124,18 @@ <h2>Decorative palettes</h2>
 		<div style="background-color: #07230d;"><span class="dl-color__hex">100</span><span class="dl-color__hue">#07230d</span></div>
 	</div>
 </div>
+
+<h2>Autogenerated palettes</h2>
+<div class="dl-color__decorative-grid">
```

Comment:
Can this be moved into its own component in `docs-components` and added to the `/libs/design/guides/color.md` readme?

Reply:
yeah I moved it but the styles are kinda messed up, can you take a look?